qiskit/pulse/__init__.py

```diff
@@ -145,8 +137,254 @@
+    pulse_prog.draw(style=style)
+
+
+In the future the pulse builder will be coupled to the
+:class:`~qiskit.circuit.QuantumCircuit` with an equivalent circuit builder
+interface.
+
+.. autosummary::
+    :toctree: ../stubs/
+
+    ~qiskit.pulse.builder.build
+
+
+Channels
+--------
+Methods to return the correct channels for the respective qubit indices.
+
+.. jupyter-execute::
+
+    from qiskit import pulse
+    from qiskit.test.mock import FakeArmonk
+
+    backend = FakeArmonk()
+
+    with pulse.build(backend) as drive_sched:
+        d0 = pulse.drive_channel(0)
+        print(d0)
+
+
+.. autosummary::
+    :toctree: ../stubs/
+
+    ~qiskit.pulse.builder.acquire_channel
+    ~qiskit.pulse.builder.control_channels
+    ~qiskit.pulse.builder.drive_channel
+    ~qiskit.pulse.builder.measure_channel
+
+
+Instructions
+------------
+Pulse instructions are available within the builder interface. Here's an example:
+
+.. jupyter-execute::
+
+    from qiskit import pulse
+    from qiskit.test.mock import FakeArmonk
+    from qiskit.visualization import SchedStyle
+
+    backend = FakeArmonk()
+
+    with pulse.build(backend) as drive_sched:
+        d0 = pulse.drive_channel(0)
+        a0 = pulse.acquire_channel(0)
+
+        pulse.play(pulse.library.Constant(10, 1.0), d0)
+        pulse.delay(20, d0)
+        pulse.shift_phase(3.14/2, d0)
+        pulse.set_phase(3.14, d0)
+        pulse.shift_frequency(1e7, d0)
+        pulse.set_frequency(5e9, d0)
+
+        with pulse.build() as temp_sched:
+            pulse.play(pulse.library.Gaussian(20, 1.0, 3.0), d0)
+            pulse.play(pulse.library.Gaussian(20, -1.0, 3.0), d0)
+
+        pulse.call(temp_sched)
+        pulse.acquire(30, a0, pulse.MemorySlot(0))
+
+    style = SchedStyle(figsize=(5, 2.5))
+    drive_sched.draw(style=style)
+
+
+.. autosummary::
+    :toctree: ../stubs/
+
+    ~qiskit.pulse.builder.acquire
+    ~qiskit.pulse.builder.barrier
+    ~qiskit.pulse.builder.call
+    ~qiskit.pulse.builder.delay
+    ~qiskit.pulse.builder.play
+    ~qiskit.pulse.builder.set_frequency
+    ~qiskit.pulse.builder.set_phase
+    ~qiskit.pulse.builder.shift_frequency
+    ~qiskit.pulse.builder.shift_phase
+    ~qiskit.pulse.builder.snapshot
+
+
+Contexts
+--------
+Builder aware contexts that modify the construction of a pulse program. For
+example an alignment context like :func:`~qiskit.pulse.builder.align_right` may
+be used to align all pulses as late as possible in a pulse program.
+
+.. jupyter-execute::
+
+    from qiskit import pulse
+    from qiskit.visualization import SchedStyle
+
+    d0 = pulse.DriveChannel(0)
+    d1 = pulse.DriveChannel(1)
+
+    with pulse.build() as pulse_prog:
+        with pulse.align_right():
+            # this pulse will start at t=0
+            pulse.play(pulse.Constant(100, 1.0), d0)
+            # this pulse will start at t=80
+            pulse.play(pulse.Constant(20, 1.0), d1)
+
+    style = SchedStyle(figsize=(5, 2.5))
```

Comment:
```suggestion
    style = SchedStyle(figsize=(3, 2), title_font_size=10, axis_font_size=8)
```
![image](https://user-images.githubusercontent.com/11485594/87706122-ff30b900-c76c-11ea-9317-e2b0a705f491.png)

Reply:
Fixed in 0194a2960.